### DISCUSSION TITLE
Ability to schedule a response (email, whatsapp,...)

### DISCUSSION THREAD

**Discussion by gitgitdown** [chatwoot/chatwoot#6952] [Feature Requests]
I'm sure that adding the option to schedule a reply would be a great asset, especially for whatsapp or messenger who don't offer this feature through their own interface.

When working with international clients, allowing you to schedule a reply for them to receive the message during their office hours is essential. For a business, the customer receiving emails or whatsapp during night hours is not ideal.
Also, it allows to prepare the next day, let's say I'm going to start work the next day at 10am, I can schedule my replies for 9:30am the day before, the next day when I start work, I've already started the discussion with the customers, it's even more ideal for team work where colleagues have different shifts. 

I hope this request will be taken into consideration and thank you for your excellent work

**Comment by glauberssoutodossantos** [comment]
+1
